### DISCUSSION TITLE
Composable components

### DISCUSSION THREAD

**Discussion by nandorojo** [tamagui/tamagui#288] [General]
Thoughts on making the `Button` a bit more composable, like so?

```tsx
<Button>
  <Button.Icon />
  <Button.Label />
</Button>
```

Technically shouldn't require breaking changes from the existing behavior, I think.

**Comment by nandorojo** [comment]
Oh I see that button text is already exported. Interesting.

**Comment by natew** [comment]
It's definitely on the roadmap, for now it's awkward.
